OpenRA.Mods.Common/Traits/Player/DeveloperMode.cs

```diff
@@ -161,10 +162,27 @@ public void ResolveOrder(Actor self, Order order)
 				case "DevGiveCash":
 				{
 					var amount = order.ExtraData != 0 ? (int)order.ExtraData : info.Cash;
+
+					Game.Debug("Giving {0} credits to player {1}", order.ExtraData == 0 ? "cheat default" : amount.ToString(CultureInfo.InvariantCulture), order.Player.PlayerName);
+
 					self.Trait<PlayerResources>().ChangeCash(amount);
 					break;
 				}
 
+				case "DevGiveCashAll":
+				{
+					var amount = order.ExtraData != 0 ? (int)order.ExtraData : info.Cash;
+					var receivingPlayers = self.World.Players.Where(p => p.Playable);
```

Comment:
Is there a reason you changed `!NonCombatant` to `Playable`?

Reply:
IMO this change makes sense. It gives cash to all the players who have slots in the lobby, but omits scripted players who may otherwise break if they unexpectedly receive or lose cash.

Reply:
This really hurts now when testing campaign AIs...